### PR TITLE
Update project to use latest Unity version (2019.2.12)

### LINKED ISSUE #3: Unity and Version Control
We should each be able to pull a base Unity project, make a simple change, and push it back for another team member to grab.

Check your name once you have verified the control flow of git works with your local Unity project repo:

- [x] John Billingham
- [x] Nicolas LaCognata
- [x] Brandy King
- [ ] Sean Simonian

### PR DESCRIPTION
**Pull request by SSimonian**
Update to Unity version 2019.2.12

closes #3

### CODE REVIEW

**Comment by MrLever** [comment]
We should actually talk about this one.

Unity does not update in place. So every update - including minor revisions - requires a new installation.

I think we should calculate when it's appropriate to update the editor as a team.

File: ProjectSettings/ProjectVersion.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.2.11f1
-m_EditorVersionWithRevision: 2019.2.11f1 (5f859a4cfee5)
+m_EditorVersion: 2019.2.12f1
+m_EditorVersionWithRevision: 2019.2.12f1 (b1a7e1fb4fa5)
```
Comment (MrLever) [inline ProjectSettings/ProjectVersion.txt]: See my comment [here](https://github.com/cop4934-fall19-group32/Project-32/pull/23#issuecomment-553278388)

**Comment by MrLever** [comment]
@j-e-b @SSimonian This pull request is high priority. Do we have an ETA?

**Comment by j-e-b** [comment]
Opens correctly with 2019.2.12. Don't know I what issue I was previously having. Anyways, merged branch.

**Pull request merged**